test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/RemoteAttributeBaseTest.cs

```diff
@@ -0,0 +1,509 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Abstractions;
+using Microsoft.AspNetCore.Mvc.DataAnnotations;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.ModelBinding.Validation;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.AspNetCore.Testing;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Localization;
+using Microsoft.Extensions.Options;
+using Moq;
+using Xunit;
+using Resources = Microsoft.AspNetCore.Mvc.ViewFeatures.Test.Resources;
+
+namespace Microsoft.AspNetCore.Mvc
+{
+    public class RemoteAttributeBaseTest
```

Comment:
I took the tests which addressed the functionality that now resides in the base attribute from RemoteAttribute tests.

Reply:
Sounds good